web/src/app/assistants/mine/AssistantsList.tsx

```diff
@@ -158,6 +174,92 @@ function AssistantListItem({
               >
                 <FiEdit2 size={16} />
               </Link>
+
+              <DefaultPopover
+                content={
+                  <div className="hover:bg-hover rounded p-2 cursor-pointer">
+                    <FiMoreHorizontal size={16} />
+                  </div>
+                }
+                side="bottom"
+                align="start"
+                sideOffset={5}
+              >
+                {[
+                  isVisible ? (
+                    <div
+                      key="remove"
+                      className="flex items-center gap-x-2"
+                      onClick={async () => {
+                        if (
+                          currentChosenAssistants &&
+                          currentChosenAssistants.length === 1
+                        ) {
+                          setPopup({
+                            message: `Cannot remove "${assistant.name}" - you must have at least one assistant.`,
+                            type: "error",
+                          });
+                          return;
+                        }
+                        const success = await removeAssistantFromList(
+                          assistant.id,
+                          currentChosenAssistants || allAssistantIds
+                        );
+                        if (success) {
+                          setPopup({
+                            message: `"${assistant.name}" has been removed from your list.`,
```

Comment:
i can't quite see what the UI is doing here until we review, but it reads as if a popup appears when adding an assistant. Sounds like it could be a bit heavyweight messaging for this operation?

Reply:
Without another notification message type here, I think a popup (really it's a toast) should be suitable for now - but agree re: having more method of communicating events in-app.